### PR TITLE
Migrate to Github API v4

### LINKED ISSUE #42: Update the GitHub service to utilize v4 of the GitHub API
#### Issue Summary

Currently the GitHub services uses v3 of the GitHub API, we would like to upgrade to v4.

#### Acceptance Criteria

* The GitHub actions [here](https://github.com/sendgrid/dx-automator/blob/development/services/github/project/api/views.py) are made by v4 GitHub API calls
* Tests have been added, proving functionality

### PR DESCRIPTION
**Pull request by nixphix**
# Closes #42

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Add support for Github API v4

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=45) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=45) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/dx-automator?pullRequest=45) it.</sub>

File: services/github/project/config.py [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 
+
 class TestingConfig(BaseConfig):
     """Testing configuration"""
     DEBUG = False
     TESTING = True
+    EXCEPTIONS = [
+        'test_exception_user'
+    ]
+    GITHUB_ORG = os.environ['GITHUB_ORG'] if os.environ['GITHUB_ORG'] else 'pallets'
```
Comment (42B) [inline services/github/project/config.py]: `os.environ('GITHUB_ORG', 'pallets')` ?

File: services/github/project/tests/test_config.py [hunk 2]
```diff
@@ -14,6 +15,7 @@ def test_app_is_development(self):
         self.assertTrue(app.config['DEBUG'] is True)
```
Comment (42B) [inline services/github/project/tests/test_config.py]: `self.assertTrue(app.config['DEBUG'])` ?

File: services/github/project/tests/test_config.py [hunk 3]
```diff
@@ -14,6 +15,7 @@ def test_app_is_development(self):
...
         self.assertFalse(current_app is None)
```
Comment (42B) [inline services/github/project/tests/test_config.py]: `self.assertIsNone(current_app)` ?

**Comment by thinkingserious** [comment]
Hello @nixphix,<br /><br/>Thanks again for the PR!<br /><br />                                         You have earned 7 out of the 5 points needed to receive glorious SendGrid Hacktoberfest swag.<br /><br />                                         Please take a moment to checkout <a href="https://dx.sendgrid.com/hacktoberfest/issues">this link</a> to find more issues to get you past the required threshold or to simply continue the celebration.<br /><br />                                         Also, <strong>please be sure you have officially registered with us <a href="https://dx.sendgrid.com/hacktoberfest/join">here</a> by November 1, 2018</strong> to qualify.<br /><br />                                         If you have any questions you can email us at <a href="mailto:dx+hacktoberfest2018@sendgrid.com">dx+hacktoberfest2018@sendgrid.com</a>.<br /><br />                                         Thank you and Happy Hacktobering!<br /><br />                                         Team SendGrid

**Pull request merged**